Summary: Remove extraneous sleep and "punt!" message.
Contributor: Alessandro Rubini <rubini@prosa.it>

Index: netcat-1.10/netcat.c
===================================================================
--- netcat-1.10.orig/netcat.c	2006-03-24 12:26:36.000000000 -0500
+++ netcat-1.10/netcat.c	2006-03-24 12:26:37.000000000 -0500
@@ -226,7 +226,6 @@
   o_verbose = 1;
   holler (str, p1, p2, p3, p4, p5, p6);
   close (netfd);
-  sleep (1);
   exit (1);
 } /* bail */
 
@@ -237,7 +236,7 @@
   errno = 0;
   if (o_verbose > 1)		/* normally we don't care */
     bail (wrote_txt, wrote_net, wrote_out);
-  bail (" punt!");
+  bail ("");
 }
 
 /* timeout and other signal handling cruft */
